File: Python/Program/Human.py
# Kelas Manusia
class Human():
    # Atribut Manusia:
    __nik = ""
    __nama = ""
    __jenis_kelamin = ""

    # Konstruktor
    def __init__(self, nik, nama, jenis_kelamin):
        self.__nik = nik
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin

    # Setter
    def set_nik(self, nik):
        self.__nik = nik

    def set_nama(self, nama):
        self.__nama = nama

    def set_jenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin

    # Getter
    def get_nik(self):
        return self.__nik

    def get_nama(self):
        return self.__nama

    def get_jenis_kelamin(self):
        return self.__jenis_kelamin

File: CPP/Program/Human.cpp
// Import file:
#include "header.hh"

// Empty konstruktor.
Human::Human()
{
    this->nik = "";
    this->nama = "";
    this->jenis_kelamin = "";
}

// Konstruktor with params.
Human::Human(string nik, string nama, string jenis_kelamin)
{
    this->nik = nik;
    this->nama = nama;
    this->jenis_kelamin = jenis_kelamin;
}

// Setter
void Human::set_nik(string nik)
{
    this->nik = nik;
}
void Human::set_nama(string nama)
{
    this->nama = nama;
}
void Human::set_jenis_kelamin(string jenis_kelamin)
{
    this->jenis_kelamin = jenis_kelamin;
}

// Getter
string Human::get_nik()
{
    return this->nik;
}
string Human::get_nama()
{
    return this->nama;
}
string Human::get_jenis_kelamin()
{
    return this->jenis_kelamin;
}

Human::~Human()
{
    
}